.github/workflows/ci.yml

```diff
@@ -1,24 +1,38 @@
 name: CI
 
-on: [push]
+on:
+  # Run once daily at 12 AM UTC, 8 PM EST
+  schedule:
+    - cron: '0 0 * * *'
+
+  # Run this on pushes to any branch
+  push:
 
 jobs:
-  ci:
-    name: Build and Test on Linux
+  unit_test:
+    name: Unit Tests
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: recursive
-      - name: Build
-        run: ./bin/build
       - name: Unit tests
         run: ./bin/test_unit
+
+  build_and_test:
```

Comment:
nit: I'm not sure `build_and_test` is descriptive enough

Reply:
Since it's just a key for the job I don't think it needs to be all that descriptive. The name provide the description and is what is seen all over.